cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -3661,17 +3711,7 @@ macro(build_google_cloud_cpp_storage)
   add_dependencies(google_cloud_cpp_dependencies nlohmann_json_ep)
   # Typically the steps to build the AWKSSDK provide `CURL::libcurl`, but if that is
   # disabled we need to provide our own.
-  if(NOT TARGET CURL::libcurl)
-    find_package(CURL REQUIRED)
-    if(NOT TARGET CURL::libcurl)
-      # For CMake 3.11 or older
-      add_library(CURL::libcurl UNKNOWN IMPORTED)
-      set_target_properties(CURL::libcurl
-                            PROPERTIES INTERFACE_INCLUDE_DIRECTORIES
-                                       "${CURL_INCLUDE_DIRS}" IMPORTED_LOCATION
-                                                              "${CURL_LIBRARIES}")
-    endif()
-  endif()
+  find_curl()
```

Comment:
There is one more `find_package(CURL)` in `build_google_cloud_cpp_storage`.

Reply:
Can we remove this `find_curl()` and the above `Typically ...` comment for cURL?